launch

```diff
@@ -189,12 +189,7 @@ def prepare_nodejs(mode):
       return False, ''
 
   install_option = []
-  ver = get_version(subprocess.check_output('npm --version', shell=True))
-  if ver[0] >= 3:
-    install_option.append('--legacy-bundling')
   pathlist = ['.',
-    'libs/brackets-server/brackets-src',
```

Comment:
Question, why installation of brackets-src isn't necessary ?

Reply:
We don't use any modules of 'bracket-src'.

Reply:
> We don't use any modules of 'brackets-src'.
May be, brackets-server uses brackets-src and it will be installed as submodule of brackets-server.
right ?

Reply:
Right. We used the source of the 'brackets-src' but we didn't use the node modules.
That line is that lets you install node modules of the 'brackets-src'.